models/doctor.py

```diff
@@ -0,0 +1,11 @@
+#!usr/bin/pyhton3
+""" A class Doctor that inherits from Staff """
+
+
+from models.staff import Staff
+from datetime import date
+
+
+class Doctor(Staff, Permissions):
```

Comment:
Why is Doctor inheriting from Permissions? Plus Permissions will be undefined in this module cos you did not import it. Just make permissions a private class attribute. Plus override job_title in this class and set it to Doctor, since that will be the title for all doctors. **job_title = "Doctor"** and **permissions = ""**. I will implement the permissions.

And please do the same for Nurse, Pharmacist, RecordOfficer and Admin classes

Reply:
I implemented this way as Permissions inherits from BaseModel which has private class attributes. So the Doctor class would inherit from BaseUser and Permissions. Since Permissions is a child of BaseModel, it will have the BaseModel attributes